mod/bookmarks/pages/bookmarks/owner.php

```diff
@@ -10,8 +10,8 @@
 	forward('', '404');
 }
 
-$title_bookmarks = elgg_echo('bookmarks:owner', array(gc_explode_translation($page_owner->title, $lang)));
-$title = elgg_echo(gc_explode_translation($page_owner->title, $lang));
+$title_bookmarks = elgg_echo('bookmarks:owner', array(gc_explode_translation($page_owner->name, $lang)));
+$title = elgg_echo(gc_explode_translation($page_owner->name, $lang));
```

Comment:
Does the `gc_explode_translation` make sense now that this is talking about a name?

Reply:
Name can be in both language